Sources/StreamChatSwiftUI/ChatChannelList/ChatChannelListView.swift

```diff
@@ -182,7 +182,7 @@ public struct ChatChannelListContentView<Factory: ViewFactory>: View {
 
     private var viewFactory: Factory
     @ObservedObject private var viewModel: ChatChannelListViewModel
-    @ObservedObject private var channelHeaderLoader = InjectedValues[\.utils].channelHeaderLoader
```

Comment:
Every time a batch of avatars was loaded, the whole channel list needed to be re-evaluated.